web/src/app/instruments/instruments-page.component.html

```
@@ -0,0 +1,62 @@
+<section class="panel">
+  <h2 class="panel-title">// watchlist.instruments</h2>
+
+  <form class="add-form" (ngSubmit)="add()">
+    <input
+      name="symbol"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)